modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java

```diff
@@ -248,6 +254,15 @@ else if (Validator.isNotNull(objectField.getRelationshipType())) {
 				return objectEntry.getTitleValue();
 			}
 		}
+		else if (Objects.equals(objectField.getType(), "Date")) {
+			Locale locale = LocaleThreadLocal.getThemeDisplayLocale();
```

Comment:
Hey @p-albuquerque , we already had the locale here using serviceContext.getLocale(), so it's better to delete this line and use this way in the next line.